File: TugasPraktikum03/Bahasa Python/PRAK301-2310817320013-AikoAnatashaWendiono.py
angka = input("Input:\n")

angka = (angka.split())

num1 = int(angka[0])
num2 = int(angka[1])

if (num1 > num2):
    num1, num2 = num2, num1
    print("Output:")
    print(num1, num2)

elif (num1 < num2):
    print("Output:")
    print(num1, num2)

File: TugasPraktikum03/Bahasa C/PRAK301-2310817320013-AikoAnatashaWendiono.c
#include <stdio.h>

int main() {
    int angka1, angka2;

    printf("Input:\n");
    scanf("%d %d", &angka1, &angka2);

    if (angka1 > angka2) {
        int angka = angka1;
        angka1 = angka2;
        angka2 = angka;
        printf("Output:\n");
        printf("%d %d\n", angka1, angka2);
        
    } else if (angka1 < angka2) {
        printf("Output:\n");
        printf("%d %d\n", angka1, angka2);
    }

    return 0;
}